tests/Support/SupportHelpersTest.php

```diff
@@ -637,6 +637,24 @@ public function testEnvEscapedString()
         $this->assertSame('x"null"x', env('foo'));
     }
 
+    public function testNestedEnvVar()
+    {
+        $_SERVER['foo'] = 'var with nested [${bar}]';
+        $_SERVER['bar'] = 'bar var';
+
+        $this->assertSame('bar var', env('bar'));
+        $this->assertSame('var with nested [bar var]', env('foo'));
+
+        unset($_SERVER['bar']);
```

Comment:
why are you unsetting the `bar` server variable and not `foo`?

or does it need to be unset at all, since you're not doing it in the other test?

Reply:
It's because from one test to another the $_SERVER variable is kept the same, so here I want to be sure that $_SERVER['bar'] is not defined.

Reply:
if SERVER variables are maintained across tests, I would recommend unsetting **all** the ones you define, to make sure your tests does not bleed into any others.

Reply:
I just did exactly the same than in the other env tests.